.github/workflows/ci.yml

```diff
@@ -0,0 +1,30 @@
+name: CI
+
+on: [push, pull_request]
```

Comment:
we might want to consider restricting running this action only on pull request, and push on specific branches (like master/main)